schemas/blocks/list_collector.json

```diff
@@ -45,6 +45,9 @@
       "add_block": {
         "$ref": "list_add_edit_remove_question.json#/block"
       },
+      "repeating_blocks": {
```

Comment:
Looking at `answer_summary.json` which has the answer_ids as a list inline, I'm vaguely thinking it would be simpler and clearer to just have that this is an array here with minItems one, referencing the repeating block, and not have a separate `list_repeating_blocks.json` file

Reply:
Great shout, done 👍